src/App.tsx

```diff
@@ -1,14 +1,81 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
+import { Filter } from './Filter';
 
 export const App: React.FC = () => {
+  const [isLoadedTodos, setIsLoadedTodos] = useState<boolean>(false);
```

Comment:
you don't need generics for primitive types
```suggestion
  const [isLoadedTodos, setIsLoadedTodos] = useState(false);
```